packages/dom/src/document.rs

```diff
@@ -87,6 +92,8 @@ pub struct Document {
     pub(crate) layout_ctx: parley::LayoutContext<TextBrush>,
 
     pub(crate) hover_node_id: Option<usize>,
+
+    pub changed: HashSet<usize>,
```

Comment:
Why is this a `HashSet` rather than a `Vec`? It never seems to be randomly queried, only ever iterated over?

Reply:
My plan was to eventually have this be used for the fine-grained accessibility tree updates but I don't see clear path for that yet 🤔 we could totally switch this to a `bool` for now if we want to stick with the coarse updates for now.

Personally I think we can get away with coarse-grained for a little while before apps in Blitz start getting overly complex

Reply:
Ah yes, `HashSet` for deduplicating insertions. Makes sense.